src/main/java/chess/controller/ApiController.java

```diff
@@ -0,0 +1,44 @@
+package chess.controller;
+
+import chess.domain.position.Position;
+import chess.dto.GameStatusDto;
+import chess.dto.MoveForm;
+import chess.dto.StatusDto;
+import chess.service.GameService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class ApiController {
+
+    private final GameService gameService;
+
+    public ApiController(GameService gameService) {
+        this.gameService = gameService;
+    }
+
+    @PatchMapping("/room/{roomId}/move")
+    public GameStatusDto moveByCommand(@PathVariable("roomId") int id, @RequestBody MoveForm moveForm) {
+        if (gameService.isEnd(id)) {
+            throw new IllegalArgumentException("게임이 종료되어 기물을 움직일 수 없습니다.");
+        }
+        gameService.move(id, Position.of(moveForm.getSource()), Position.of(moveForm.getTarget()));
+        return new GameStatusDto(gameService.isEnd(id));
+    }
+
+    @GetMapping("/room/{roomId}/status")
+    public StatusDto showStatus(@PathVariable("roomId") int id) {
+        return gameService.status(id);
+    }
+
+    @PostMapping("/room/{roomId}/end")
+    public ResponseEntity<Void> endGame(@PathVariable("roomId") int id, @RequestParam("password") String password) {
+        if (!gameService.isEnd(id)) {
+            throw new IllegalArgumentException("진행중인 게임은 삭제할 수 없습니다.");
+        }
+        if (!gameService.end(id, password)) {
+            throw new IllegalArgumentException("게임 삭제에 실패하였습니다.");
+        }
```

Comment:
```suggestion
        gameService.end(id, password);
```
컨트롤러에 로직을 두면 다른 View에서 이 로직을 재활용할 때 복붙을 해야되는 상황이 발생할 거 같아요 😄 
Game#canDelete 또는 Game#canEnd 정도를 만들어서 서비스에서 체크하면 좋을 것 같습니다.